src/RestApiClient.js

```diff
@@ -74,7 +74,8 @@ class RestApiClient {
    * @return {String}
    */
   getLocalUrl() {
-    return `//${window.location.hostname}${window.location.port}`;
+    const port = window.location.port ? `:${window.location.port}` : '';
```

Comment:
It does! But we need to append the `:` at the beginning if it's not an empty string.